### ISSUE TITLE
Add ability to execute stateless data plane commands

### ISSUE THREAD

**Issue by craicoverflow** [redhat-developer/app-services-cli#685]
Currently you must `use` a Kafka cluster before operating on the data plane. It should be possible to perform these commands without needing to switch context:

```shell
rhoas kafka topic list --instance-id=293443044305459569
```

```shell
rhoas kafka topic list --instance-name=endas-kafka
```

Up for discussion: cc @rkpattnaik780 @wtrocki

**Comment by wtrocki** [comment]
This is done already as instance-id is being used in topics, consumer-groups and registry.

@rkpattnaik780 can you close this?

**Comment by craicoverflow** [comment]
This is only done for the service-registry commands.

**Comment by wtrocki** [comment]
We need to add support for instance-id to:

```
rhoas kafka topic * --instance-id
rhoas kafka acls * --instance-id
rhoas kafka consumer-groups * --instance-id
```

Example how the flag can be added is here:
https://github.com/redhat-developer/app-services-cli/blob/main/pkg/cmd/kafka/acl/flagutil/flagset.go#L206-L215

Example how instance-ids are handled in existing commands

https://github.com/redhat-developer/app-services-cli/blob/main/pkg/cmd/registry/artifact/crud/get/get.go#L80:L80

**Comment by wtrocki** [comment]
Seems to be done.

**Comment by rkpattnaik780** [comment]
Fixed by #1483
